src/index.js

```diff
@@ -1,14 +1,28 @@
 const express = require("express");
+const bodyParser = require("body-parser");
+const { ValidationError } = require("express-validation");
+
 const app = express();
 const routes = require("../src/api/routes/index");
 
 require("../src/models/index");
 
 app.use(express.json({}));
 app.use(express.urlencoded({ extended: true }));
+app.use(bodyParser.json());
 
```

Comment:
Nowadays, bodyParser is no longer necessary in express application because json parser is inbuilt in express so line 10 is enough;